springwolf-plugins/springwolf-cloud-stream-plugin/src/test/java/io/github/springwolf/plugins/cloudstream/asyncapi/scanners/channels/CloudStreamFunctionChannelsScannerIntegrationTest.java

```diff
@@ -451,5 +451,11 @@ public Function<String, Integer> testFunction() {
         public Function<KStream<Void, String>, KStream<Void, Integer>> kStreamTestFunction() {
             return stream -> stream.mapValues(s -> 1);
         }
+
+        @Bean
+        @GooglePubSubSchemaSetting(encoding = "BINARY", name = "project/test")
+        public Consumer<String> testPubSubConsumer() {
+            return System.out::println;
+        }
```

Comment:
This test configuration is pulled into every spring boot test

I assume that you only want to have a method that can be used in the CloudStreamFunctionChannelsBindingSchemaSettingIntegrationTest.
A more lightweight approach has been to define a bean within the test class itself, as done: SpringAnnotationMethodLevelChannelsScannerIntegrationTest

The scanner might need some adaption to allow scanning for a specific method (untested)
```java
@Override
    public Map<String, ChannelObject> scan() {
        List<Map.Entry<String, ChannelObject>> channelObjects = beanMethodsScanner.getBeanMethods().stream().flatMap(this::scan).toList();
        return ChannelMerger.mergeChannels(channelObjects);
    }

    public Stream<Map.Entry<String, ChannelObject>> scan(Method method) {
        return functionalChannelBeanBuilder.fromMethodBean(method).stream()
                .filter(this::isChannelBean)
                .map(this::toChannelEntry);
    }
```